database/dbSeed.js

```diff
@@ -11,15 +18,35 @@ const seedPropertyDetails = function() {
     'Bed and breakfast': ['Farm stay', 'Nature lodge'],
     'Boutique hotel': ['Boutique hotel', 'Hostel', 'Resort', 'Serviced apartment']
   };
-  
   let spaceAvail = ['Private room', 'Entire place', 'Shared room'];
+  let adjectives = ['huge', 'extraordinary', 'amazing', 'pleasant', 'cozy', 'enjoyable', 'delightful', 'charming', 'beautiful', 'fine', 'pleasing', 'elegant', 'alluring', 'lovely', 'attractive', 'chic', 'enchanting', 'irresistible', 'stimulating' ];
+  let attraction = ['park', 'amusement park', 'beach', 'campsite', 'shopping outlet', 'farmers market', 'resort', 'famous trail', 'night life plaza', 'winery', 'golf course', 'dog park', 'rose garden', 'museum', 'zoo', 'recreation area'];
 
-  db.connection.query('INSERT INTO Property_details (property_type, property_narrow_type, property_space_available, dedicated_guest_space, registered_business, min_length_stay, max_length_stay, guest_capacity, rooms, bathrooms) VALUES ("Apartment", "Loft", "Shared room", true, false, 2, 5, 10, 5, 2)', (err, result)=>{
-    if (err) {
-      throw err;
-    }
-    console.log(result);
-  });
+  while (propertyCount > 0) {
+    let typeArr = Object.keys(propType);
+    let type = typeArr[Math.floor(Math.random() * 6)];
+    let min = Math.ceil(Math.random() * 6);
+    let narrow = propType[type][Math.floor(Math.random() * propType[type].length)];
```

Comment:
nice!

Reply:
may be worth considering variable name here that's representative of the content, i.e. typeName or typeValue